src/Raven.Studio/typescript/viewmodels/database/advanced/databaseIDs.ts

```diff
@@ -58,24 +59,32 @@ class databaseIDs extends shardViewModelBase {
                 this.isForbidden(!accessManager.default.isOperatorOrAbove());
                 
                 if (this.isForbidden()) {
-                    deferred.resolve({can: true});
+                    deferred.resolve({ can: true });
                 } else {
-                    const fetchStatsTaskArray = this.fetchAllStatsTasks();
-                    const fetchUnusedIDsTask = this.fetchUnusedDatabaseIDs();
-
-                    $.when<any>(...fetchStatsTaskArray, fetchUnusedIDsTask)
-                        .then(() => {
-                            this.usedIDs(this.usedIDsArray);
-                            this.idsFromCVs(this.idsFromCVsSet);
-                            deferred.resolve({can: true});
-                        })
-                        .fail(() => deferred.resolve({redirect: appUrl.forStatus(this.db)}));
+                    this.loadData()
+                        .then(() => deferred.resolve({ can: true }))
+                        .catch(() => deferred.resolve({ redirect: appUrl.forStatus(this.db) }));
```

Comment:
https://issues.hibernatingrhinos.com/issue/RavenDB-18393 we will handle that in this issue